### DISCUSSION TITLE
Top Driver - Ayaz Ullah. How am I able to be no.1 most days?

### DISCUSSION THREAD

**Discussion by Magizhan** [nammayatri/nammayatri#302] [Show and tell]
### How to stay on top? -   _from our top driver_ Ayaz Ullah

Below are some of the screenshots shared by Ayaz Ullah after he finishes his day. He has been on the top of the leader board consistently for a long time. How is he able to do this ?  Let us understand from him directly

<img width="1310" alt="Screen Shot 2023-03-24 at 8 36 02 PM" src="https://user-images.githubusercontent.com/34025961/227563387-87fec0d7-acf6-48c8-bccf-67d9e6d56a78.png">

**NY:**  How is it possible for you to do so many rides every day using Namma Yatri? 

**Ayaz:** Simple sir. 
- **I use only Namma Yatri always and I don't use other apps. I** have deleted the other apps.  I get more money for the same or less work because of zero commission when compared to other apps
- I don't add +10, +20 unnecessarily and so the customer don't cancel my trips. 
- I accept most of the trips when I'm online and nowadays, I get enough trips for the day. 

**NY:** How are you able to use only Namma Yatri always? In certain locations, you might not even get ride requests. 

**Ayaz:** I have figured out where all I'll get more Namma Yatri requests by now. **Sometimes, I travel back 2-4kms without a ride too** to get another Namma Yatri ride. Effectively, this is fine for me because this happens only at certain times and I'm better than not taking any ride or taking a very ineffective ride. 

**NY:** Any feedback for improving Namma Yatri? 

**Ayaz:** I love Namma Yatri. Whenever there is an issue, I reach out to the team and get it resolved immediately. I would **recommend all my auto driver friends to make the best use of an app like Namma Yatri Driver**. Thank You.

**Comment by vimalkumar** [comment]
Can we do these?

1. Do a proper video interview with him and share it in our youtube channel & send it to our registered drivers.

2. Create a video - a day in Ayaz's work. It can even be a fast forward version of the auto icon moving in a map... the earnings counter increasing with time moving in fast forward. Show that the earnings are more with less time spent. 

3. Create a leaderboard and share it with the drivers. It can be a part of our open data dashboard. It can have links to the above story.
